reboot-stability-check

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+start="${start:-"01"}"
+runs="${runs:-"30"}"
+host="${1:?"Need 'host' to check reboot stability for. Can be multiple hosts"}"
+boot_timeout="${boot_timeout:-"600"}"
+ping_timeout="${ping_timeout:-"30"}"
+sleep_time="${sleep_time:-"120"}"
+
+check-one-host() {
+    echo -n "run: $run, $host: ping .. "
+        timeout -k 5 "$boot_timeout" sh -c "until ping -c$ping_timeout $host >/dev/null; do :; done"
```

Comment:
Is this picking up the individual `$host` from the loop in `main`?

I would find it clearer to use different variables eg. call it `$hosts` elsewhere

Reply:
ok, done